### PR TITLE
docs(flutter): Document Flutter web limitations

### PR DESCRIPTION
**Pull request by denrase**
## DESCRIBE YOUR PR

Add some Flutter web limitations to the features page.

Based on https://github.com/getsentry/sentry-dart/issues/897
Relates to https://github.com/getsentry/sentry-docs/issues/4312

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/flutter/features/index.mdx [hunk 1]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
     - as well as `http` with the [Dart SDK](/platforms/dart/).
 - Integrations for sqflite, routing and more. For a complete list, see our [integrations list](/platforms/flutter/integrations/).
 - [Release Health](/product/releases/health/) tracks crash free users and sessions.
-- Limited support for Flutter Web.
+
+**Web Limitations:**
+
+Sentry has limited support for Flutter Web, but there are several limitations compared with mobile platforms:
```
Comment (buenaflor) [inline docs/platforms/flutter/features/index.mdx:28]:
```suggestion
Sentry has support for Flutter Web, but there are several limitations compared with mobile platforms:
```

File: docs/platforms/flutter/features/index.mdx [hunk 2]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+
+- Issue titles are not human readable, as they show the minified name.
+- Stack traces are not symbolicated when compiling using WASM.
```
Comment (buenaflor) [inline docs/platforms/flutter/features/index.mdx:31]:
```suggestion
- Stack traces are not symbolicated when compiling using WebAssembly (WASM).
```

**Review by buenaflor — APPROVED** [review]
Looks good!

File: docs/platforms/flutter/features/index.mdx [hunk 3]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+- Stack traces are not symbolicated when loaded as a Chrome extension.
```
Comment (kahest) [inline docs/platforms/flutter/features/index.mdx:32]:
```suggestion
- Stack traces are not symbolicated when loaded as a browser extension.
```

File: docs/platforms/flutter/features/index.mdx (hunk 1 above)
Comment (kahest) [inline docs/platforms/flutter/features/index.mdx:28]:
```suggestion
Sentry supports Flutter Web as well, with the following limitations:
```

File: docs/platforms/flutter/features/index.mdx [hunk 4]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
     - as well as `http` with the [Dart SDK](/platforms/dart/).
 - Integrations for sqflite, routing and more. For a complete list, see our [integrations list](/platforms/flutter/integrations/).
 - [Release Health](/product/releases/health/) tracks crash free users and sessions.
-- Limited support for Flutter Web.
+
+**Web Limitations:**
+
+Sentry supports Flutter Web as well, with the following limitations:
+
+- Issue titles are not human readable, as they show the minified name.
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:30]:
```suggestion
- Issue titles are not human-readable, as they show the minified name.
```

File: docs/platforms/flutter/features/index.mdx [hunk 5]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+- Stack traces are not symbolicated when compiling using WebAssembly (WASM).
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:31]:
```suggestion
- Stack traces are not symbolicated when compiled using WebAssembly (WASM).
```

File: docs/platforms/flutter/features/index.mdx [hunk 6]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+- Stack traces are not symbolicated when loaded as a browser extension.
+- No offline caching for exceptions
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:33]:
```suggestion
- Offline caching is not supported for exceptions.
```

File: docs/platforms/flutter/features/index.mdx [hunk 7]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+- No support for [Release Health](/product/releases/health/)
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:34]:
```suggestion
- [Release Health](/product/releases/health/) is not supported.
```

File: docs/platforms/flutter/features/index.mdx [hunk 8]
```diff
@@ -22,4 +22,14 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+- Device context data is not as rich as on mobile
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:35]:
```suggestion
- Device context data is not as rich on web as it is on mobile.
```

**Review by vivianyentran — APPROVED** [review]
Added some copyedits. Thanks for adding this!

**Comment by buenaflor** [comment]
we need a vercel preview run pls 🙏

**Pull request merged**